### PR TITLE
Add last_ttl variable

### PR DESCRIPTION
**Pull request by foxdalas**
If "No After Date" < last_ttl
Generate certificate!

### CODE REVIEW

**Comment by zonorti** [comment]
lgtm

File: resources/gencert.rb [hunk 1]
```diff
@@ -125,6 +130,24 @@ def decoded_shared_key
...
+    ca_cert_host = ::File.read(ca_path)
+    ca_cert_server = ca
+
+    if (Time.now.to_i+last_ttl) >= not_after.to_time.to_i || ca_cert_host != ca_cert_server
+      Chef::Log.warn("Certificate #{cert} expire: #{not_after}")
+      Chef::Log.warn("Host CA certificate: #{ca_cert_host}")
+      Chef::Log.warn("Server CA certificate: #{ca_cert_server}")
+      return false
+    else
+      Chef::Log.info("Certificate #{cert} is ok")
```
Comment (dshmelev) [inline resources/gencert.rb]: I think its no needs.

**Comment by foxdalas** [comment]
fixed

**Comment by dshmelev** [comment]
squash please =)

**Pull request merged**